### DISCUSSION TITLE
Should Axmol Engine having a "Donate" button like Godot have?

### DISCUSSION THREAD

**Discussion by aismann** [axmolengine/axmol#1582] [General]
https://fund.godotengine.org/

Don't say No! 
Donation is free and a gift for some hart work. 

Another reason is:
```
I think I can speak for all 'Top 10' contributors:
We try to fix as fast as possible.

But we make it for free and mostly for having fun.
But each detected bug/issue can also be on a roadmap and not fixed as fast as possible.

Off topic:
Maybe a small donation is a possible way to push some wishes on the top.
But this is not really easy to handle and not a good way.
I have seen such requests on lot of 'open source' frameworks, also on `Cocos2d-x` or `Godot`.
```

Maybe this gift is enough:
If Axmol Engine helped you please consider adding a star on GitHub. This means a lot to the authors.

**Comment by rh101** [comment]
Donating to a project isn't as simple as it may seem.  It often requires the formal creation of an organization (such as a non-profit) registered in a jurisdiction, and the donations would go to it, where it would require someone to manage it.  It's not a cheap process, both in terms of time and money.

> Maybe this gift is enough:
> If Axmol Engine helped you please consider adding a star on GitHub. This means a lot to the authors.

Yes, that would probably be the best way to support the project.

**Comment by halx99** [comment]
Not sure does this works: https://opencollective.com/axmol

EDIT: I have test $1 (username: deal11), seems it works

**Reply by rh101** [reply]
I didn't realise that service existed. It seems like a great way to get this going.

**Reply by aismann** [reply]
A permanent link (Wiki?) is usefull now .
The big feature is all can see who has donate.

**Reply by halx99** [reply]
Yes

**Reply by halx99** [reply]
I have no idea where to put it

**Comment by aismann** [comment]
> I have no idea where to put it

I will do. Done!
https://github.com/axmolengine/axmol/wiki/Donate-for-Axmol%3F-Yes-you-can!

**Comment by aismann** [comment]
Close it here.
